instrumented/nightly-tests/src/androidTest/kotlin/com/datadog/android/nightly/rum/RumResourceTrackingE2ETests.kt

```diff
@@ -113,6 +114,44 @@ internal class RumResourceTrackingE2ETests {
         launch(ResourceTrackingFirstPartyHostsActivity::class.java)
     }
 
+    /**
+     * apiMethodSignature: com.datadog.android.core.configuration.Configuration$Builder#fun useViewTrackingStrategy(com.datadog.android.rum.tracking.ViewTrackingStrategy?): Builder
+     * apiMethodSignature: com.datadog.android.rum.tracking.ActivityViewTrackingStrategy#constructor(Boolean, ComponentPredicate<android.app.Activity> = AcceptAllActivities())
+     * apiMethodSignature: com.datadog.android.tracing.TracingInterceptor#constructor(TracedRequestListener = NoOpTracedRequestListener(), Float = DEFAULT_TRACE_SAMPLING_RATE)
+     * apiMethodSignature: com.datadog.android.DatadogInterceptor#constructor(com.datadog.android.tracing.TracedRequestListener = NoOpTracedRequestListener(), com.datadog.android.rum.RumResourceAttributesProvider = NoOpRumResourceAttributesProvider(), Float = DEFAULT_TRACE_SAMPLING_RATE)
+     * apiMethodSignature: com.datadog.android.rum.RumInterceptor#constructor(List<String> = emptyList(), RumResourceAttributesProvider = NoOpRumResourceAttributesProvider(), Float = DEFAULT_TRACE_SAMPLING_RATE)
+     * apiMethodSignature: com.datadog.android.core.configuration.Configuration$Builder#fun setFirstPartyHostsWithHeaderType(Map<String, Set<com.datadog.android.tracing.TracingHeaderType>>): Builder
+     */
+    @Test
+    fun rum_resource_tracking_with_first_party_hosts_and_header_types() {
+        // In this case because the Span is generated by the backend as "android.request" Span
+        // there is no metric automatically generated for it so we had to create a special
+        // custom metric: "rum_resource_tracking_with_first_party_hosts" to assert this behaviour
```

Comment:
this test will add more hits to the existing `rum_resource_tracking_with_first_party_hosts` metric, triggering this monitor "[RUM] [Android] Nightly - rum_resource_tracking_with_first_party_hosts: Number of spans generated from resource tracking should not be greater than 2/day as we only have one use case that should generate this Span".

I think we should either modify the value in this monitor, or create a separate metric for this test (and update the previous metric, so that it doesn't cover the new one as well).